lib/Experian/IDAuth.pm

```diff
@@ -205,21 +202,15 @@ sub _build_country_code_tag {
     my $two_digit_country   = $self->{residence};
     my $three_digit_country = uc Locale::Country::country_code2code($two_digit_country, LOCALE_CODE_ALPHA_2, LOCALE_CODE_ALPHA_3);
 
-    if (not $three_digit_country) {
-        warn "Client " . $self->{client_id} . " could not get country from residence [$two_digit_country]";
-        return;
-    }
+    $three_digit_country or croak "Client " . $self->{client_id} . " could not get country from residence [$two_digit_country]";
```

Comment:
normally I'd put the condition at the end, `croak '...' unless $three_digit_country`, since I find it makes the action clearer (compare to a sentence with no verb in English).